examples/run_CPMG.py

```diff
@@ -159,10 +151,11 @@
     ["ph2", "ph_diff", "nEcho", "t2"],
     [len(ph2), len(ph_diff), config_dict["nEchoes"], -1],
 )
-data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis(
-    "ph2", ph2 / 4
-).setaxis("ph_diff", ph_diff / 4)
-data.set_prop("postproc_type", "spincore_diffph_SE_v2")
+data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis("ph2", ph2 / 4).setaxis(
+    "ph_diff", ph_diff / 4
+)
+data.set_units("t2", "s")
+data.set_prop("postproc_type", "spincore_CPMG_v2")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore